timechart_launcher/main.py

```diff
@@ -15,18 +15,50 @@
 from timechart.displays.main_display import TimeChartDisplay
 
 
+try:
```

Comment:
Why do you need this code twice in here and at line 30?

Reply:
Some keyboard shortcuts I pressed didn't take effect. Fixed.